scripts/install_opencv.sh

```diff
@@ -1,32 +1,87 @@
 #!/bin/bash
 
-git clone --branch 3.4.6-instrumented https://github.com/ILLIXR/opencv "${temp_dir}/opencv"
-git clone --branch 3.4.6 https://github.com/opencv/opencv_contrib/  "${temp_dir}/opencv_contrib"
-if ! nvcc --version &> /dev/null
-then
-    echo "nvcc not found"
-else
-    echo "nvcc found"
+#--- Script for installing OpenCV ---#
+
+
+## Source the default values for imported variables
+. scripts/default_values.sh
+
+## Source the global helper functions
+. scripts/system_utils.sh
+
+## Source the configurations for our dependencies
+. deps.sh
+
+
+### Package metadata setup ###
+
+if [ -z "${dep_name}" ]; then
+    dep_name="${dep_name_opencv}"
+fi
+
+if [ -z "${src_dir}" ]; then
+    src_dir="${parent_dir_opencv}/${dep_name_opencv}"
+fi
+
+if [ -z "${dep_ver}" ]; then
+    dep_ver="${dep_ver_opencv}"
+fi
+
+dep_ver_extra="3.4.6"
+repo_url="https://github.com/ILLIXR/opencv"
+repo_url_extra="https://github.com/opencv/opencv_contrib"
+opencv_dir_extra="${opt_dir}/opencv_contrib"
+build_dir="${src_dir}/build"
+file_cleanup_list="a.out cmake_hdf5_test.o"
+
+flags_cuda="-allow-unsupported-compiler"
```

Comment:
Sorry, forgot to update you earlier. We no longer require CUDA when installing OpenCV (good riddance). All of the CUDA related stuff in the OpenCV install can be safely removed.